rfcs/text/0002_encrypted_attributes.md

```diff
@@ -0,0 +1,228 @@
+- Start Date: 2019-03-22
+- RFC PR: https://github.com/elastic/kibana/pull/34526
+- Kibana Issue: (leave this empty)
+
+# Summary
+
+In order to support the action service we need a way to encrypt/decrypt
```

Comment:
I'd rather not explicitly couple this to actions even in the summary language. "In order to support storing secrets in saved objects" or something like that